packages/block-library/src/social-links/editor.scss

```diff
@@ -40,7 +40,7 @@
 	// Wrap the remaining placeholders in a container so the plus can overlap.
 	> .wp-block-social-links__social-placeholder-icons {
 		display: flex;
-		position: absolute;
+		//position: absolute;
```

Comment:
When ready, just delete instead of commenting out. I can understand when iterating and keeping track of changes.